src/backend/DBHelpers.h

```diff
@@ -99,9 +130,30 @@ struct NFTsData
         , ledgerSequence(meta.getLgrSeq())
         , transactionIndex(meta.getIndex())
         , owner(owner)
+        , uri({})
```

Comment:
This should not be needed, uri is empty optional by default.
If you really want to explicitly set it please prefer `std::nullopt`.